### PR TITLE
Logio kibana ui

### PR DESCRIPTION
**Pull request by brittainhard**
Pane for showing both logio and kibana on the crawl page.

### CODE REVIEW

**Comment by brittainhard** [comment]
So I think this is ready? Let me to some more manual testing and I'll let you know.

**Comment by brittainhard** [comment]
@ahmadia is it ok to merge this?

File: source/apps/crawl_space/templates/crawl_space/crawl.html [hunk 1]
```diff
@@ -219,11 +220,25 @@
...
+                <li role="presentation" class="active"><a href="#logio" aria-controls="logio" role="tab" data-toggle="tab">log.io</a></li>
+                <li role="presentation"><a href="#kibana" aria-controls="kibana" role="tab" data-toggle="tab">Kibana</a></li>
+            </ul>
+        </div>
+        <div class="tab-content">
+            <div role="tabpanel" class="tab-pane active" id="logio">
+                <iframe src="http://127.0.0.1:28778" style="width:100%;height:600px;border:1px solid #EEEEEE;"></iframe>
+            </div>
+            <div role="tabpanel" class="tab-pane" id="kibana">
+                <iframe src="http://127.0.0.1:9999" style="width:100%;height:600px;border:1px solid #EEEEEE;"></iframe>
```
Comment (ahmadia) [inline source/apps/crawl_space/templates/crawl_space/crawl.html]: can you turn these hardcodes into variable names:

127.0.0.1:9999 - Kibana endpoint?
127.0.0.1:28778 - log.io server endpoint?

Comment (ahmadia) [inline source/apps/crawl_space/templates/crawl_space/crawl.html]: I'm guessing you could use a Jinja template variable here to do this.

Comment (brittainhard) [inline source/apps/crawl_space/templates/crawl_space/crawl.html]: I could do that, but I don't really have anywhere to persist that data right now. Is this a suggestion or do you need this?

**Comment by brittainhard** [comment]
Merging if no comment.

**Pull request merged**